app/src/main/res/layout/fragment_contacts.xml

```diff
@@ -11,14 +11,32 @@
         android:layout_height="wrap_content"
         app:title="@string/title_contacts" />
 
-    <com.yterletskyi.happyfriend.common.view.SearchView
-        android:id="@+id/etSearch"
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginHorizontal="8dp"
-        android:layout_marginVertical="8dp"
-        android:hint="@string/search_hint_contacts"
-        android:inputType="text" />
+        android:layout_height="wrap_content">
+
+        <com.yterletskyi.happyfriend.common.view.SearchView
+            android:id="@+id/etSearch"
+            android:layout_width="match_parent"
```

Comment:
please use `0dp`, setting `match_parent` for views inside ConstraintLayout is not correct